crates/wit-component/src/printing.rs

```diff
@@ -1,11 +1,16 @@
 use anyhow::{anyhow, bail, Result};
 use std::fmt::{self, Write};
+use std::mem;
 use wit_parser::*;
 
 /// A utility for printing WebAssembly interface definitions to a string.
 #[derive(Default)]
 pub struct WitPrinter {
     output: Output,
+
+    // Count of how many items in this current block have been printed to print
+    // a blank line between each item, but not the first item.
+    items: usize,
```

Comment:
Would this be simpler as a `any_items: bool`, which just records whether we've seen zero or more-than-zero? When I saw this was a `usize`, I scanned through the code to see if it was ever decremented or compared with a number other than zero.

Reply:
Ah yes that's much clearer!